test/plugin/scenarios/pulsar-scenario/config/expectedData.yaml

```diff
@@ -97,6 +99,7 @@ segmentItems:
             - { key: transmission.latency, value: not blank }
             - { key: mq.broker, value: not blank }
             - { key: mq.topic, value: test }
+            - { key: mq.body, value: "1" }
```

Comment:
This is how the message is built, 
```java
producer.newMessage().key("testKey").value(Integer.toString(1).getBytes()).property("TEST", "TEST").send();
```

But you only set `1` as body? Could you explain?

Reply:
This 1 was set by the previous submitter, and I only tested the corresponding expected value based on this value. If need test message bodies in other formats, I can modify and resubmit them.

Reply:
But there is not only value. There are a key and properties. aren't those message body too?